docs/sources/enterprise/license/license-restrictions.md

```diff
@@ -12,135 +12,41 @@ When you become a Grafana Enterprise customer, you receive a license that govern
-
-### Grant dashboard edit without save permissions
 
-You can grant viewers the ability to edit but not save a dashboard. This setting is useful when you have users that want to explore dashboards, but only need the generated view temporarily and do not require an editor/admin license.
+### Determine the number of active users
 
-> This setting does not affect viewers’ license role billing.
-
-To grant dashboard edit without save permissions:
-
-1. Open the `grafana.ini` file that contains default configurations.
-
-   The name and location of the file depends on your installation. For information about locating
-   the default configurations file, refer to [Configuration]({{< relref "../../administration/configuration.md" >}}).
-
-1. Change the `viewers-can-edit` setting to `true`.
-1. Restart Grafana Enterprise.
-
-### Determine the number of active users for each licensed role
-
-Because you can assign a user global permissions while also enhancing (or limiting) their access to specific dashboards, it might be helpful for you to periodically check license counts.
-
-To determine the number of active users for each licensed role:
+To determine the number of active users:
 
 1. Sign in to Grafana Enterprise as a System Administrator.
 
 1. Click **Server Admin** (the shield icon).
 
 1. Click **Statistics and licensing**.
 
-1. Review the role utilization count on the **Utilization** panel.
-
-> Utilization data updates when the user signs back in to Grafana after you change their permissions.
-
-### Optimize your tiered license
-
-Grafana counts viewer users separately from editor/admin users. If you reach the limit imposed by your license for either of these user types, use the following techniques to make optimum use of the licenses you purchased.
-
-- **Review and update organization permissions.** You can modify organization permissions to reduce the count of one type of user. For example, if you reach the limit of editor/admin users, assign viewer permissions to one (or more) editor/admin users who agree to read-only access to dashboards. The license count updates when a user signs back in to Grafana.
-- **Review and update dashboard permissions.** Because viewers can also count toward editor/admin usage, if they are granted access to edit specific dashboards, review your dashboard permissions and adjust accordingly. Remove editor or admin permissions where necessary.
-- **Delete users.** Review the number of users you have for each type of permission and remove inactive users. Removing users immediately frees up space for new users.
-
-> When you delete a user you also delete their preferences, dashboard and folder permissions, and references to that user from the dashboard versions they have updated.
-
-Users who are synced via SAML, Oauth, or LDAP will also lose these attributes.
+1. Review the utilization count on the **Utilization** panel.
 
-### Update user permissions for dashboards
```

Comment:
Should there be a note pointing to RBAC, which provides the ability to change the access certain roles of users have to specific dashboards regardless of license type, to replace this functionality being deprecated by tiered licenses?

Reply:
We talked about permissions here before because it had a strong impact on the active user count that was used for  licensing. As it has no impact anymore on licensing, I don't think it's relevant to keep it here anymore.